std/range/package.d

```diff
@@ -3515,8 +3515,26 @@ private alias lengthType(R) = typeof(R.init.length.init);
    offers random access iff all ranges offer random access, and also
    offers mutation and swapping if all ranges offer it. Due to this, $(D
    Zip) is extremely powerful because it allows manipulating several
-   ranges in lockstep. For example, the following code sorts two arrays
-   in parallel:
+   ranges in lockstep.
+   For example, the following code sorts two arrays in parallel:
```

Comment:
agreed - just didn't know a nice way to fit it into the text. I will try again ;-)